collectd-consul/README.md

```diff
@@ -0,0 +1,372 @@
+
+    [<img src='./img/chart_time_to_append_entries.png' width=200px>](./img/chart_time_to_append_entries.png)
+
+  - **Number of RPC queries**: Total number of rpc queries per interval. This is a general measure of all read volume.
+
+    [<img src='./img/chart_number_of_rpc_queries.png' width=200px>](./img/chart_number_of_rpc_queries.png)
+
+  - **Cluster Joins and Leaves**: This chart tracks successful node joins and leaves in the Serf memberlist. 
+
+    [<img src='./img/chart_join_leave.png' width=200px>](./img/chart_join_leave.png)
+
+  - **Leader time to reconcile**: Shows the time it takes for the leader to reconcile Serf membership and what is reflected in Consul's store.
+
+    [<img src='./img/chart_leader_reconcile.png' width=200px>](./img/chart_leader_reconcile.png)
+
+  - **Serf Events**: Consul provides an event feature by which custom events can be propagated across your entire datacenter. This chart shows the number of events processed by Consul agents per interval. Using this chart you can track if triggered events were processed by a consul node. Additinally, you can also easily setup a chart to track events for a selected node in the CLIENT and SERVER dashboard.
+
+    [<img src='./img/chart_serf_events.png' width=200px>](./img/chart_serf_events.png)
+
+  - **Serf Event Queue**: Shows the avg and max number of backlog of serf events in queue of Consul agents.
+
+    [<img src='./img/chart_serf_event_queue.png' width=200px>](./img/chart_serf_event_queue.png)
+
+- **CONSUL CLIENT**
+
+  - **Number of allocated heap objects**: Gives the number of heap objects allocated to the consul process. Indicates memory pressure on a Consul node.
+
+    [<img src='./img/chart_heap_objects.png' width=200px>](./img/chart_heap_objects.png)
+
+  - **Allocated Bytes**: Number of allocated bytes to the Consul process.
+  
+    [<img src='./img/chart_allocated_bytes.png' width=200px>](./img/chart_allocated_bytes.png)
+
+  - **Number of GO routines**: The number of GO routines Consul is running. This is a general load pressure indicator for Consul agent.
+
+    [<img src='./img/chart_go_routines.png' width=200px>](./img/chart_go_routines.png)
+
+  - **Network Latency**: Shows the avg, max and min network latency between the node and other nodes in the datacenter.
+
+    [<img src='./img/chart_network_latency.png' width=200px>](./img/chart_network_latency.png)
+
+  - **Time to service DNS queries**: Consul provides both DNS and HTTP interfaces for service discovery. This shows the time it takes to service forward and reverse DNS lookups by the selected node.
+
+    [<img src='./img/chart_dns_queries.png' width=200px>](./img/chart_dns_queries.png)
+
+- **CONSUL SERVER**
+All charts metioned in the Client dashboard are also present in the Server dashboard. In addition to those, below charts are also present -
```

Comment:
"In addition to those, the following charts are present"  (also leave off the "-").